src/components/AudioPlayer.vue

```diff
@@ -1,87 +1,122 @@
-            <v-list-item-content>
-              <v-list-item-title v-html="sourceFile.title" />
-              <v-list-item-subtitle>
-                <span v-html="sourceFile.author" />
-                <span class="ml-4">{{ timer }} / {{ duration }}</span>
-              </v-list-item-subtitle>
-            </v-list-item-content>
-
-            <v-spacer />
-
-            <v-list-item-icon :class="{ 'mx-5': $vuetify.breakpoint.mdAndUp }">
-              <v-icon
-                v-if="$data.error"
-                :title="$data.error"
-              >
-                mdi-alert
-              </v-icon>
-              <v-btn
-                icon
-                v-else-if="loading"
-                loading
-              />
-              <v-btn
-                v-else-if="playing"
-                icon
-                @click="pause"
-              >
-                <v-icon>
-                  mdi-pause
-                </v-icon>
-              </v-btn>
-              <v-btn
-                v-else
-                icon
-                @click="play"
+          <v-scroll-y-reverse-transition>
+            <v-list>
+              <v-list-item
+                dark
               >
-                <v-icon>
-                  mdi-play
-                </v-icon>
-              </v-btn>
-            </v-list-item-icon>
+                <v-list-item-avatar
+                  tile
+                  size="60"
+                >
+                  <v-img
+                    height="60"
+                    aspect-ratio="1"
+                    bac
+                    gradient="to bottom, rgba(255,255,255,.1), rgba(255,255,255,.5)"
+                    :src="sourceFile.src"
+                  >
+                    <v-icon
+                      size="32"
+                      color="white"
+                      style="opacity: 0.3;
+                        position: absolute;
+                        top: 50%;
+                        left: 50%;
+                        transform: translate(-50%, -50%);"
+                    >
+                      mdi-music
+                    </v-icon>
+                  </v-img>
+                </v-list-item-avatar>
+                <v-list-item-content>
+                  <v-list-item-title v-html="sourceFile.title" />
+                  <v-list-item-subtitle>
+                    <span v-html="sourceFile.author" />
+                    <span>{{ timer }} / {{ duration }}</span>
+                  </v-list-item-subtitle>
+                </v-list-item-content>
 
-            <v-list-item-icon
-              class="ml-0"
-              :class="{ 'mr-3': $vuetify.breakpoint.mdAndUp }"
-            >
-              <v-btn
-                icon
-                @click="closePlayer"
-              >
-                <v-icon>mdi-close</v-icon>
-              </v-btn>
-            </v-list-item-icon>
-          </v-list-item>
-        </v-list>
+                <div
+                  class=""
```

Comment:
Perhaps best to remove empty class, as it doesn't do anything.

Reply:
Done